platform/broadcom/sonic-platform-modules-alphanetworks/snj60b0-320f/modules/snj60b0-320f_sfp.c

```diff
@@ -0,0 +1,2068 @@
+/*
+ * SFP driver for alphanetworks snj60b0-320f sfp

```

Comment:
Please use existing optoe driver in lieu of creating a custom driver.

Reply:
Because of HW design, we use a custom driver.

Reply:
Can you explain why your HW design does not work with the optoe driver? As long as your HW conforms to the standard specs, it should be able to access the EEPROM fine.

Reply:
Why we need proprietary EEPROM driver is because we use “CPLD” instead of I/O port expansion in our HW design. Therefore we need to control CPLD first, then we could access EEPROM. (The CPLD will also have functionality other than EEPROM access.)
It is our HW design for saving cost and space.

Reply:
@lguohan to comment on this.

Reply:
@lguohan , Please help to comment this.

Reply:
@lguohan , Please help to comment this.

Reply:
@lguohan , Please help to comment this.

Reply:
@lguohan,  Please help to comment it?